smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -161,4 +699,37 @@ public void roomDestroyed(MultiUserChat alternateMUC, String reason) {
         assertEquals(0, muc.getOccupantsCount());
         assertNull(muc.getNickname());
     }
+
+    /**
+     * Gets a random room name
+     *
+     * @param prefix A prefix to add to the room name for descriptive purposes
+     */
+    private EntityBareJid getRandomRoom(String prefix) throws XmppStringprepException {
+        final String roomNameLocal = String.join("-", prefix, testRunId, StringUtils.insecureRandomString(6));
+        return JidCreate.entityBareFrom(Localpart.from(roomNameLocal), mucService.getDomain());
+    }
+
+    /**
+     * Destroys a MUC room, ignoring any exceptions.
+     *
+     * @param muc The room to destroy (can be null).
+     */
+    static void tryDestroy(final MultiUserChat muc) {
+        if (muc == null) {
+            return;
+        }
+        try {
+            muc.destroy("test fixture teardown", null);
+        } catch (Exception ignore) {
+        }
```

Comment:
Always try to catch the most specific exception(s). Furthermore, no empty `catch` blocks. Either log the exceptions, with `FINEST` level if you want, or rethrow. I am currently updating ErrorProne in Smack, and with the new version, there the new [EmptyCatch bug pattern](https://errorprone.info/bugpattern/EmptyCatch) will be active.

Reply:
I've added the log line. Good call.